dashboard/models.py

```diff
@@ -41,11 +41,12 @@ class Meta:
         abstract = True
 
     @classmethod
-    def user_qset(cls, user, program=None):
+    def user_qset(cls, user, program=None, provider='edxorg'):
```

Comment:
I think maybe we should split this into 2 methods so our code is more explicit about when edx-only is being called:

```python
    @classmethod
    def user_qset(cls, user): ...

    @classmethod
    def program_qset(cls, program, provider): ...
```

So existing calls of `user_qset()` would remain as such and other calls could turn into `user_qset().program_qset(program, provider)`.

Reply:
Thanks! I updated this, not the way you suggested though, but I think it makes sense for the usages of these method. Let me know what you think.